python/sglang/srt/managers/cache_controller.py

```diff
@@ -577,7 +577,8 @@ def terminate_prefetch(self, operation):
     def append_host_mem_release(self, host_indices: torch.Tensor):
         chunks = host_indices.split(self.mem_pool_host.page_size)
         for chunk in chunks:
-            self.host_mem_release_queue.put(chunk)
+            if len(chunk) > 0:
+                self.host_mem_release_queue.put(chunk)
```

Comment:
should we just do a check on `host_indices`? that would be easier and less operations

Reply:
Updated @xiezhq-hermann